cfme/tests/infrastructure/test_host_drift_analysis.py

```diff
@@ -40,109 +37,83 @@ def pytest_generate_tests(metafunc):
     testgen.parametrize(metafunc, argnames, new_argvalues, ids=new_idlist, scope="module")
 
 
-@pytest.mark.meta(blockers=[1242655])
-def test_host_drift_analysis(appliance, request, setup_provider, provider, host, soft_assert):
+@pytest.fixture(scope='module')
+def a_host(host, appliance, provider):
+    host_collection = appliance.collections.hosts
+    return host_collection.instantiate(name=host.name, provider=provider)
+
+
+@pytest.fixture(scope='module')
+def set_host_credentials(provider, a_host, setup_provider_modscope):
+    host_list = provider.hosts
+    host_names = [host.name for host in host_list]
+    for host_name in host_names:
+        host_data = [host for host in host_list if host.name == host_name][0]
+        a_host.update_credentials_rest(credentials=host_data.credentials)
+
+    yield
+    a_host.update_credentials_rest(credentials=Host.Credential(principal="", secret=""))
```

Comment:
**QUESTION**
I still don't understand the point to not use https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/hosts.py#L21